adr/proposed/StatementSeperators.md

```diff
@@ -0,0 +1,111 @@
+# Overview
+
+Every SQL dialect has a way to separate:
+
+1. Statements
+2. Batches
+
+In most SQL dialects, the statement separator is a semi-colon (';'). In some SQL dialects, the statement separator can be dynamically configured.
```

Comment:
do you intend to add the possibility to change it in FM ?

Reply:
yes